2009-03-21  Michael Tautschnig  <mt@debian.org>

	* bin/faireboot: sleep for 10 seconds before leaving faireboot script
		(closes: #520554)
Index: trunk/bin/faireboot
===================================================================
--- trunk.orig/bin/faireboot
+++ trunk/bin/faireboot	
@@ -36,4 +36,6 @@
     killall -CONT rcS
     killall rcS
     init 6 &
+    echo "Waiting for shutdown to complete"
+    sleep 10
 fi
